includes/wf_crm_webform_postprocess.inc

```diff
@@ -1513,6 +1513,23 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
         'entity_table' => 'civicrm_contribution',
       );
     }
+    // LineItems
+    $fid = "civicrm_1_lineitem_1_contribution_line_total";
+    if (isset($this->enabled[$fid])) {
+      foreach ($this->data['lineitem'][1]['contribution'] as $n => $lineitem) {
+        $fid = "civicrm_1_lineitem_{$n}_contribution_line_total";
+        if ($this->getData($fid) > 0) {
+          $this->line_items[] = array(
+            'qty' => 1,
```

Comment:
We could potentially add a field for quantity.

Reply:
It crossed my mind too! But then we'd probably also have to store the priceset in CiviCRM tables - as people will really think these are pricesets now?